plugin/route53/alias_resolver_test.go

```diff
@@ -0,0 +1,82 @@
+package route53
+
+import (
+	"context"
+	"fmt"
+	"github.com/miekg/dns"
+	"net"
+	"testing"
+	"time"
+)
+
+func getFreePort() (int, error) {
```

Comment:
not needed, pkg/test or upper level test should be added.

Reply:
The authoritative resolver is not exposed, so not easily testable from the pkg/test package.

The code is actually quite simple don't know if this even warrants a test. It just queries the authoritative name-server for the records.

What do you think @miekg ?

Reply:
Not sure, but this contraption should not be added, the current test code should be usable

Reply:
I'll remove this then.